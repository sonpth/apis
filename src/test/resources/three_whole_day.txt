2022-06-01T00:00:00 43
2022-06-01T00:30:00 51
2022-06-01T01:00:00 63
2022-06-01T01:30:00 44
2022-06-01T02:00:00 87
2022-06-01T02:30:00 80
2022-06-01T03:00:00 1
2022-06-01T03:30:00 93
2022-06-01T04:00:00 73
2022-06-01T04:30:00 60
2022-06-01T05:00:00 69
2022-06-01T05:30:00 3
2022-06-01T06:00:00 92
2022-06-01T06:30:00 38
2022-06-01T07:00:00 93
2022-06-01T07:30:00 0
2022-06-01T08:00:00 18
2022-06-01T08:30:00 3
2022-06-01T09:00:00 54
2022-06-01T09:30:00 34
2022-06-01T10:00:00 54
2022-06-01T10:30:00 63
2022-06-01T11:00:00 75
2022-06-01T11:30:00 2
2022-06-01T12:00:00 36
2022-06-01T12:30:00 97
2022-06-01T13:00:00 13
2022-06-01T13:30:00 18
2022-06-01T14:00:00 35
2022-06-01T14:30:00 25
2022-06-01T15:00:00 13
2022-06-01T15:30:00 50
2022-06-01T16:00:00 99
2022-06-01T16:30:00 88
2022-06-01T17:00:00 54
2022-06-01T17:30:00 27
2022-06-01T18:00:00 53
2022-06-01T18:30:00 26
2022-06-01T19:00:00 81
2022-06-01T19:30:00 55
2022-06-01T20:00:00 41
2022-06-01T20:30:00 87
2022-06-01T21:00:00 3
2022-06-01T21:30:00 58
2022-06-01T22:00:00 71
2022-06-01T22:30:00 22
2022-06-01T23:00:00 98
2022-06-01T23:30:00 21
2022-06-02T00:00:00 14
2022-06-02T00:30:00 58
2022-06-02T01:00:00 36
2022-06-02T01:30:00 10
2022-06-02T02:00:00 41
2022-06-02T02:30:00 96
2022-06-02T03:00:00 98
2022-06-02T03:30:00 51
2022-06-02T04:00:00 86
2022-06-02T04:30:00 88
2022-06-02T05:00:00 77
2022-06-02T05:30:00 6
2022-06-02T06:00:00 101
2022-06-02T06:30:00 74
2022-06-02T07:00:00 27
2022-06-02T07:30:00 86
2022-06-02T08:00:00 31
2022-06-02T08:30:00 56
2022-06-02T09:00:00 8
2022-06-02T09:30:00 9
2022-06-02T10:00:00 46
2022-06-02T10:30:00 60
2022-06-02T11:00:00 47
2022-06-02T11:30:00 100
2022-06-02T12:00:00 14
2022-06-02T12:30:00 74
2022-06-02T13:00:00 17
2022-06-02T13:30:00 0
2022-06-02T14:00:00 1
2022-06-02T14:30:00 0
2022-06-02T15:00:00 10
2022-06-02T15:30:00 45
2022-06-02T16:00:00 18
2022-06-02T16:30:00 32
2022-06-02T17:00:00 24
2022-06-02T17:30:00 16
2022-06-02T18:00:00 23
2022-06-02T18:30:00 88
2022-06-02T19:00:00 56
2022-06-02T19:30:00 82
2022-06-02T20:00:00 43
2022-06-02T20:30:00 26
2022-06-02T21:00:00 99
2022-06-02T21:30:00 38
2022-06-02T22:00:00 8
2022-06-02T22:30:00 90
2022-06-02T23:00:00 9
2022-06-02T23:30:00 20
2022-06-03T00:00:00 98
2022-06-03T00:30:00 12
2022-06-03T01:00:00 58
2022-06-03T01:30:00 2
2022-06-03T02:00:00 32
2022-06-03T02:30:00 94
2022-06-03T03:00:00 95
2022-06-03T03:30:00 31
2022-06-03T04:00:00 51
2022-06-03T04:30:00 47
2022-06-03T05:00:00 24
2022-06-03T05:30:00 18
2022-06-03T06:00:00 57
2022-06-03T06:30:00 12
2022-06-03T07:00:00 48
2022-06-03T07:30:00 43
2022-06-03T08:00:00 20
2022-06-03T08:30:00 36
2022-06-03T09:00:00 97
2022-06-03T09:30:00 42
2022-06-03T10:00:00 46
2022-06-03T10:30:00 59
2022-06-03T11:00:00 19
2022-06-03T11:30:00 19
2022-06-03T12:00:00 13
2022-06-03T12:30:00 43
2022-06-03T13:00:00 102
2022-06-03T13:30:00 55
2022-06-03T14:00:00 31
2022-06-03T14:30:00 10
2022-06-03T15:00:00 97
2022-06-03T15:30:00 97
2022-06-03T16:00:00 0
2022-06-03T16:30:00 24
2022-06-03T17:00:00 23
2022-06-03T17:30:00 59
2022-06-03T18:00:00 7
2022-06-03T18:30:00 18
2022-06-03T19:00:00 29
2022-06-03T19:30:00 36
2022-06-03T20:00:00 40
2022-06-03T20:30:00 65
2022-06-03T21:00:00 43
2022-06-03T21:30:00 82
2022-06-03T22:00:00 29
2022-06-03T22:30:00 85
2022-06-03T23:00:00 78
2022-06-03T23:30:00 65