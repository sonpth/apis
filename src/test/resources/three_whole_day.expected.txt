2022-06-01 2364
2022-06-02 2139
2022-06-03 2191
